### ISSUE TITLE
AdminLTE 2.0

### ISSUE THREAD

**Issue by almasaeed2010** [almasaeed2010/AdminLTE#209]
Hello Everyone,

AdminLTE 2.0 is on its way with great new features and many improvements to the current code. We decided to ask you to post your concerns and issues with the current version. Also, what would make AdminLTE better? Post your suggestions and concerns here and we will try our best to make it a reality. 

We also wanted to note that AdminLTE 2.0 will ship with a premium collection of widgets and customized plugins that will contribute greatly to revolutionize your experience with the template and widen the range of its uses. The premium collection will be available for an affordable price (starting at $5 for a regular license). We would appreciate it if you let us know what you'd like to add to AdminLTE to make it better.

Early sneak peak at AdminLTE 2.0 new dashboard
![screen shot 2015-01-17 at 1 49 31 pm](https://cloud.githubusercontent.com/assets/1512664/5789948/bc813680-9e4f-11e4-8e3f-4ac0d0a600ec.png)

List of customized plugins that we plan to add for an affordable price:
- File upload
- Form wizard
- Customized dashboard for blogs
- Customized dashboard for e-commerce
- Profile page
- x-editable plugin
- Animations
- Customized Widgets for:
  - Profile
  - News feed
  - Weather
- Right sidebar to serve as a chat, notifications and quick stats menu.
- Right to Left support
- Free updates once purchased for any premium plugins changes

Let us know if there is anything else that you'd like to see in AdminLTE 2.0. We are open to new ideas! 

Thank you! We are eager to hear from you.

**Comment by almasaeed2010** [comment]
To clarify, AdminLTE 2.0 will still be an open source free template. We would hate to disappoint our fans! So, free updates and new things will still be posted regularly as away of saying thank you to all of our contributors and users.

**Comment by kressly** [comment]
Hi 
I can't see the Right sidebar to serve as a chat, notifications and quick stats menu on the image. I am interested in that.

Skin Management
1- And also i am interested in a skin based on each of collor inside the buttons page like on this picture so that with php we can give the ability for every visitor to either choose one of those skin or customize them. If at least we can have a skin based on all these colors, that will be awesome.
![1](https://cloud.githubusercontent.com/assets/925132/5790060/93c6d58a-9e7f-11e4-93a1-e6946783a342.png)

2- I believe AdminLte should not loose it's original blue color because to me that is one of the main thing that makes it stand out among any other design. So for version 2, it is great to have a black sidebar at the left BUT if we can still have a skin with the white (at the left side) like the blue original skin, that will also be great. 
![1](https://cloud.githubusercontent.com/assets/925132/5790110/63677370-9e81-11e4-9097-8d68d344bf5d.png)


Black is great but not most people are fan of black but people are fan of blue like Facebook, twitter, LinkedIn, AdminLte.. :+1: 
Now it will be very awesome if what we have for the blue, we can have something similar with the color used for .bg-maroon,  ..bg-orange, .bg-fuchsia, bg-red, .bg-yellow, ..

File Upload
The file Upload will be great if it enables to add facial recognition like some free jquery script that are available, image cropping

Profile
1 - It will be great if we can have a responsive cover image and profile picture on it like facebook or even better with other things like maybe the ability to move the profile image and fix it anywhere on the cover image
2- The cover image and the profile picture will both be able to be use as a slide (CLICK left or right and other images may appear and the left and righ buttons may appear only onmouse over like slimscrool's scrool)

And of course i guess all will still be responsive.

Of course you are not obliged to implement all these. And there are things that we will ask in private that will never be made public as soon as some of our project brings fruits.


I am sure our friend the good programmer and design @lenamtl has things to add

**Comment by almasaeed2010** [comment]
Those are great suggestions. If you think if anything else, don't hesitate to let us know. And although we might not be able to do everything on the list in such a short period of time, all suggestions make our priorities clearer to us. This way, we deliver what people need first and add to it later. 

Thank you @kressly. And we hope to hear from everyone else too.

**Comment by lenamtl** [comment]
A nice too have Language selector in the top menu.

Could you clarified what you mean by premium plugins, any example:
*Free updates once purchased for any premium plugins changes*

**Comment by almasaeed2010** [comment]
The premium plugins are the customized plugins I listed above. They will be offered with a license with the following prices:
- **Single License ($5).** For use in one product.
- **Multiple License ($15).** For use in unlimited number of projects.
- **Extended License ($50).** For use in unlimited large projects that are offered for sale.

More information about the plugins and the licenses will be available on the website.

Up to this point, AdminLTE 2.0 has these new modifications:
- Major layout bug fixes
- Change in layout mark up
- Added transitions to sidebar collapse
- New skins and modified previous skins
- Change in color scheme to a more complementing scheme 
- Added footer support
- Removed pace.js from the main app.js
- Added support for collapsed sidebar as an initial state (add .sidebar-collapse to the body tag)
- Added boxed layout (.layout-boxed)
- Enhanced consistency in padding and margining
- Updated Bootstrap to 3.3.2 
- Fixed navbar dropdown menu on small screens positioning issues.
- Updated Ion Icons to 2.0.0
- Updated FontAwesome to 4.2.0
- Added ChartJS 1.0.1

**Comment by kressly** [comment]
Le language selector will be great especially for those of us who speak french and other languages. But it will be good if it can also be removed easily if we do not want to add it to a particular project.
I do have a question. How will the license be used ? Does it mean that some of the code will now be encrypted ?

**Comment by almasaeed2010** [comment]
No code will be decrypted. The demo will show the premium items but in order to download them one should buy them from the website. In other words, they won't be pushed to this repository.

**Comment by kressly** [comment]
Oh oK :+1:

**Comment by kressly** [comment]
AminLte should also be tested in all major Browsers especially in Mozilla and IE, Safari and Chrome. I can see AdminLte has well been tested in Chrome because @almasaeed2010 is a Chrome user but it will be nice if we can also test it in Mozilla as well as IE

**Comment by kressly** [comment]
Indepedent Classes
I think AdminLte will far greater if we were to have indepedent classes. What i mean is that when for example in the AdminLte.css we have a code like `.right-side >  .content-header {` ,  if  i were to add anything like an id example `<span id='id_du_milieu'></span>` between right-side and content-header, it will affect content-header and i will have to rewrite the css like `.right-side > #id_du_milieu > .content-header {` 
But if the classes where to be independent, when i apply `class='content-header'` to an element, whether it is under right-side or not it will work and will reduce my load of work drastically .

**Comment by lenamtl** [comment]
Nice to have more navigation option:
Sticky top menu
Sticky footer 
On small device drop-down menu instead of pull menu
Footer
Right menu panel (the same as the left menu but located at the right)
Mega menu

**Comment by awsp** [comment]
+1 @kressly
It would be very nice to have a skin-oriented AdminLTE. Currently there is only 2 skins available.  

I am in particular concern the folder structure, especially when, 
  - developing new skins
  - choosing a particular skin only to minimize the size of the files :D

```
/less/  
  /skins/
    /light_blue/
      /settings.less <-- contains all settings / vars from this skin, ex: color definitions, font, etc.
      /dropdown.less <-- skin-specific dialog
      /buttons.less
      /...
    /light_purple/
      /settings.less <-- contains all settings / vars from this skin, ex: color definitions, font, etc.
      /...
  /dropdown.less <-- contains general css rules of a dialog
  /buttons.less
  /...
```

Also, I notice it is sometimes hard to sort of taking controls on some of the css settings as it is already defined in bootstrap. So maybe it's a good idea to add a main css class at DOM root to allow easy control. 
For instance,
```html
<body class="adminlte skin-blue">
  ...
</body>
```

```less
// general 
.adminlte {
  a { text-decoration: none }  
}

// skin_blue 
.adminlte {
  .form-control { padding: 2px 6px; } // override bootstrap value without putting !important

  &.skin-blue {
    a { color: @light-blue }
    .form-control { border: 1px solid @light-blue }
  }
  &.skin-red {
    a { color: @red; }
    .form-control { border: 1px solid @red }
  }
}
```


Maybe I am wrong, but I seem not seeing the modals and windows in AdminLTE (besides the original bootstrap one? I guess), so it would be great to have a AdminLTE version of dialogs, message boxes , etc. 

Finally, a tree view would also be great too.

**Comment by almasaeed2010** [comment]
@lenamtl @kressly @awsp Thank you all so much! Great suggestions!

I'd like to note that we are focusing currently on the code architecture and a few major issues with the initial design consistency. For example, not all elements have a consistent padding and margining, which is critical for a professional appearance. For the code architecture, here is an excerpt of the new app.js file: 
```js
/*! AdminLTE app.js
 * ================
 * Main JS application file for AdminLTE v2. This file
 * should be included in all pages. It controls some layout
 * options and implements exclusive AdminLTE plugins.
 *
 * @Author  Almsaeed Studio
 * @Support <http://www.almsaeedstudio.com>
 * @Email   <support@almsaeedstudio.com>
 * @version 2.0
 * @license MIT <http://opensource.org/licenses/MIT>
 */

'use strict';

/* AdminLTE
 *
 * @type Object
 * @description $.AdminLTE is the main object for the template's app.
 *				It's used for implementing functions and options related
 *				to the template. Keeping everything wrapped in an object
 *				prevents conflict with other plugins and is a better
 *				way to organize our code.
 */
$.AdminLTE = new Object();

/* --------------------
 * - AdminLTE Options -
 * --------------------
 * Modify these options to suit your implementation
 */
$.AdminLTE.options = {
  //Add slimscroll to navbar menus
  //This requires you to load the slimscroll plugin
  //in every page before app.js
  navbarMenuSlimscroll: true,
  navbarMenuSlimscrollWidth: "3px", //The width of the scroll bar
  navbarMenuHeight: "200px", //The height of the inner menu
  //Activate sidebar push menu
  sidebarPushMenu: true,
  //Activate sidebar slimscroll if the fixed layout is set
  sidebarSlimScroll: true,
  //BoxRefresh Plugin
  enableBoxRefresh: true,
  //Bootstrap.js tooltip
  enableBSToppltip: true,
  BSTooltipSelector: "[data-toggle='tooltip']",
  //Box Widget Plugin. Enable this plugin
  //to allow boxes to be collapsed and/or removed
  enableBoxWidget: true,
  //Box Widget plugin options
  boxWidgetOptions: {
    boxWidgetIcons: {
      //The icon that triggers the collapse event
      collapse: 'fa fa-minus',
      //The icon that trigger the opening event
      open: 'fa fa-plus',
      //The icon that triggers the removing event
      remove: 'fa fa-times'
    },
    boxWidgetSelectors: {
      //Remove button selector
      remove: '[data-widget="remove"]',
      //Collapse button selector
      collapse: '[data-widget="collapse"]'
    }
  },
  //Define the set of colors to use globally around the website
  colors: {
    lightBlue: "#3c8dbc",
    red: "#f56954",
    green: "#00a65a",
    aqua: "#00c0ef",
    yellow: "#f39c12",
    blue: "#0073b7",
    navy: "#001F3F",
    teal: "#39CCCC",
    olive: "#3D9970",
    lime: "#01FF70",
    orange: "#FF851B",
    fuchsia: "#F012BE",
    purple: "#8E24AA",
    maroon: "#D81B60",
    black: "#222222",
    gray: "#eaeaec"
  }
};
```
The code goes on from there to implemented the requested features by the developer(s) without forcing them to use unwanted plugins. We also have gone through every less file and modified the code to match Bootstrap's guide line. We think that this is going to contribute greatly to enhancing the developers' experience with AdminLTE.

Finally, lots of new widgets and elements are being constructed. However, before creating new elements, we are trying to make every Bootstrap component redesigned to be consistent with AdminLTE's design.

We hope the new techniques we are applying would bring a solid and premium feel and look to the open source AdminLTE. We are really excited to show off the changes, which brought the idea of releasing a pre-release (alpha and beta releases) to get a quick feedback before releasing the final production version. We will let you know when the pre-release is ready to be viewed and downloaded.

We greatly appreciate your contributions!

**Comment by almasaeed2010** [comment]
@kressly we got a virtual machine setup so we can test AdminLTE in all browsers including IE9+, FF, Chrome, and safari (on iphones and android devices too). Nonetheless, there is always the possibility that we miss something along the way, so by all means don't hesitate to report any issues if any were found.

**Comment by biapar** [comment]
--:))

**Comment by kressly** [comment]
@almasaeed2010 Ok and really u did a great job. I am turning the whole thing into a very dynamic thing. Thanks a lot.

**Comment by ircsasw** [comment]
Default status "collapsed-box" in ".box"

**Comment by almasaeed2010** [comment]
@kressly  @lenamtl  
Progress of AdminLTE 2.0 is now being daily uploaded to [Almsaeed Studio Themes](almsaeedstudio.com/themes/AdminLTE). Feel free to explore the new features as they are being created. Once the alpha version is ready, I will push it to this repo.

@ircsasw Thanks! This feature has been added to AdminLTE 2.0.

**Comment by kressly** [comment]
This is what i see in Mozilla latest version
![1](https://cloud.githubusercontent.com/assets/925132/5890623/6745d848-a45c-11e4-8bf4-8ecea685c964.png)
I will suggest that you make AdminLte version 2 completely independent of Cdn because the certificate issued by maxcdn for example is causing a problem with firefox. And due to that only those using Chrome or other browser will be able to see the product. Imagine a new user coming to the page and instead of seing the page he sees that image, he will never know how good the work is.
So if possible, make direct link this time meaning, incorporate bootstrap, jquery and all pluging in the download so that it will stop people from telling this or this does not work.

Now from Chrome, it works because Chrome does not have much issues with certificate. 
Having the two dashboard give us a choice and the possibility to create our own third or personalize dashboard :+1: 

The original blue skin we all fell in love with and some of us are still in love with is no more part of the system :-1: 
On dashboard 2 the visitor report is not displaying when you move into a responsive mod :-1: 

The Chat box on the Dashboard two is awesome. It helps differenciate between a chat issue by a user from the chat issue by another  :+1:  :+1: 
However, they should be a collapsible box to display the list of people the member is already chatting with so that he can easily move from one chat with one person to the other
![3](https://cloud.githubusercontent.com/assets/925132/5890668/a8fd57dc-a45e-11e4-92e9-ed2490fb17c2.png)
The message Box on the top should have the ability to disapear on click even if there is no # or link so that an ajax request can easily be perform on it. It can be a class
The General UI alerts and callout are good :+1:  . However we should add the previous one to it for new comers and also for the possibility to have many choice. Having a choice, free will is the key of many success.
Choice between boxed layout or not :+1: 
Version 2 is greatly expected with a profile display. Most people wonder how AdminLte Profile page will look like .
Because of time i have not tested in IE yet.
Finaly, you should give some bonus for my contributions Lol :+1:

**Comment by kressly** [comment]
I guess still because of the cdn it does not display well on windows phone 8.1 IE . However it does on Windows IE

**Comment by almasaeed2010** [comment]
Hello @kressly 
Let me start off by thank you for the contributions and kind compliments! 

I have tested the template on many browsers (Mozilla, Chrome, Safari, and Opera) and it is working properly on all of them. The screenshot of Mozilla that you posted seem to be having trouble getting the bootstrap.css from the CDN, which could mean that there was some down time with their server. We would like to have everything not pulled from CDNs anymore but that might slow down our host significantly with the current host plan we have. However, we are upgrading to a better host soon, which will let us be able to remove all CDNs. For the time being, AdminLTE 2.0 alpha will ship with CDNs but the production release will contain all the files needed to run it. 

Regrading IE on Windows Phone, take a look at [Bootstrap (IE10 in Windows 8 and Windows phone 8 issue.)](http://getbootstrap.com/getting-started/#support-ie10-width) We will have to add that to our CSS and JS since bootstrap doesn't do it. 

The chat widget, and the rest of the template for that matter, are incomplete yet. This version is just to view progress and keep the excitement going :). The Direct Chat Widget has many surprises that will be available in 2.0 alpha including a contacts pane as you mentioned :+1: 

The last thing is the visitors report. I am still debating allowing the sparkline charts to display in extra small devices because it is not displaying properly on 320px wide screens. Nonetheless, I think I can figure something out for it.

**Comment by anoldguy** [comment]
I was just about to start modifying the less files to add another skin so it fit better with the branding we have. What's the best way to future-proof changes like that on my end?

I'd hate to put in many hours customizing a skin, only to have to re-implement in v2.  Also, is there a build script to regenerate the distributable CSS after modifying the less files?

Those questions aside, I like how v2 is looking!

**Comment by kressly** [comment]
@almasaeed2010  :+1: 
That really keeps the excitement going ..
 We can't wait to have version 2.

**Comment by kasperhartwich** [comment]
Great work. Looking forward to the release of the new version. When will it be able to see it in git?

I have some feedback. 

1. I would love to see some example screens to present a lot of data, f.ex. a customer overview with orders or just a screen with realy much data presented in a pretty way. 

2. When on a large screen, the menu or container does not expand full height:
![screen shot 2015-01-26 at 13 31 25](https://cloud.githubusercontent.com/assets/616231/5899695/bfe6781e-a55f-11e4-93c2-e40eca4bb077.png)

Thank you again. Is there set a deadline for a release?

**Comment by almasaeed2010** [comment]
@anoldguy The structure of the sidebar and header didn't change. The only thing you might want to consider is that the header now has the class .main-header instead of .header. So, even if you created a style for v1 it should work on v2 with a slight modification to the sidebar selector. V2 will ship with a grunt.js file that watches changes to less files and automatically compiles them. And thanks for the compliment!

@kasperhartwich Since I am still a student my time is limited so I won't promise a certain date for the release but I will say that it looks like the alpha release will be available for testing and playing with soon. Regarding the example page, I am not sure how to do that without having the data :-z. I will look into it though.
In your screenshot it seems like the jQuery script have not loaded. Probably because I forgot to update that page. 

Thank you all for the great suggestions. You'll hear from me soon.

**Comment by gunaseln** [comment]
First of all, this is an awesome framework, and it really suits our needs very well. The attention to detail is simply mind blowing. I played with your demo, and have a question about the coverage for the following features in your latest release:

=> Mailbox - is there a way to open, and read emails? The demo doesn't open a mail, rather shows the tabular view of messages listed. I see you are out of time, but it would be very helpful if it's possible to have a complete set of mail program specific features - like opening a mail, uploading attachments, downloading attachments, embedding multi-media

=> Ours is an ASP .NET MVC project - I am still integrating it, and am not completely done with it. Do you see any reason to suspect there could be issues w/ ASP .NET MVC? One issue I saw is the LESS files throwing warnings during the pre-process compilation stage. I later downloaded koala and pre-processed the files to get rid of the BOM error

=> If this is not available, that's okay - may be you can add this to your backlog.
Thanks

**Comment by almasaeed2010** [comment]
@gunaseln 
Regarding the mailbox, that's a feature that has been requested several times and we think it's important too. We will for sure add it with the next release (asap).

To be honest, I have never used ASP.NET but I saw the template being used with ASP in many projects. They always had and issue with a form that comes after <body>. I have fixed this issue in 2.0. 

The LESS files shouldn't produce any errors if you are using the latest LESS compiler. 

Thank you for the suggestions and hopefully you'll see results soon :+1:

**Comment by gunaseln** [comment]
Thank you for the quick response - I played with your stuff last night, and you are really creative and extremely talented.


Date: Wed, 28 Jan 2015 08:07:07 -0800

**Comment by almasaeed2010** [comment]
NEW

checkout the contacts pane in the direct chat widget by clicking on the chat bubbles icon at the top.

@kressly <-- that was your suggestions :+1:

**Comment by ssrahman** [comment]
Hi, Is there any possibility of main Horizontal navigation?

**Comment by almasaeed2010** [comment]
@ssrahman could you provide an example? It would help assess the difficulty of creating the element.

**Comment by ssrahman** [comment]
We created an admin theme. You can see the Horizontal Menu: http://demo.netfunda.com/admin/

**Comment by almasaeed2010** [comment]
@ssrahman That's not difficult to create since the navbar is part of Bootstrap. In fact, all the components are in AdminLTE. I will create an example page that uses this method soon. Thanks for the suggestion!

**Comment by kasperhartwich** [comment]
Hi. Again thank you for the great work. I will be looking forward to buy the premium version.
Regarding a info page, i would suggest looking into building a 'profile' page for the theme like a profile page in Facebook/LinkedIn where different information about contact, education aso. would be.

Will you make 2.0 available via github?

**Comment by almasaeed2010** [comment]
Hello @kasperhartwich 
Thank you for you interest in AdminLTE! The profile page is under development and will be available with the premium collection. 
AdminLTE will stay open source and will be available on Github. However, a release date has not been set yet. The alpha release should be out soon though.

**Comment by kasperhartwich** [comment]
Sounds great. For open-source and premium edition releasedates, are we talking weeks, months or christmas? :-/

**Comment by almasaeed2010** [comment]
@kasperhartwich 
Open source I'd say 1 week and a 2 to 3 weeks for premium collection (1.0). And by christmas we'll have even more fun components to play with :)

**Comment by kressly** [comment]
@almasaeed2010  :+1:

**Comment by kasperhartwich** [comment]
Sounds incredible! Looking forward to your work! And again, thank you!!

**Comment by kressly** [comment]
Please for the box close to the chat box, can you add some 1 or 2 icon like you did for the message envelop ? So that if there is a new message from a member, we can use ajax to show the number of new message from that member that has not been read .

Thanks in advance Genius Designer @almasaeed2010

**Comment by lenamtl** [comment]
Does it display ok if your not using the CDN (put the files on the sever)?

**Comment by almasaeed2010** [comment]
@ssrahman how does that look?
http://almsaeedstudio.com/themes/AdminLTE/pages/layout/top-nav.html

**Comment by JCKodel** [comment]
Better support for metro Internet Explorer 11, so we can use drag and drop on metro/phone gap apps. 

Would be veeeery nice if all of adminlte2 worked as angular directives

**Comment by almasaeed2010** [comment]
AdminLTE 2.0 alpha is now available on Github. You can download the source from Github or this [production version](http://almsaeedstudio.com/Themes/AdminLTE/AdminLTE2-dist.zip) which doesn't contain any development files.

Looking forward to hear your feedback.

**Comment by lenamtl** [comment]
Thanks

**Comment by lenamtl** [comment]
FYI your Google analytics script is in the package...

**Comment by almasaeed2010** [comment]
@lenamtl thanks for letting me know. I have removed it.

**Comment by kressly** [comment]
@almasaeed2010 
I just downloaded it and really i Love the Dashboard 2. I will keep it and release later as an update after delivering the AdminLte 1 to my customer. Especially the chat box :+1: 
Latest member Box :+1: 
Jquery Knob Inline chart :+1: 

We are still using Font awesome Icon however they do not work on most android browser with opera i have tested them on. They display a question mark :-1: 

The name of the member and his picture on top left :+1: 

New blank page style :+1: 
Possibility of choosing :+1: 
Copyright  :+1: 
MultiLevel :+1:  :+1:  :+1: 

Labels ??? 

Documentation :+1:  :+1:  :+1:  :+1:  :+1:  :+1: :+1:  :+1:  :+1: 

However i am yet to test on mobile devices
Great Job bro

**Comment by kressly** [comment]
Wooow Oh My God i have not seen the different skins . Just saw it while testing on my windows phone. Plus 100 bro. Woooow

**Comment by lenamtl** [comment]
@kressly I really like font awesome I'm using it all the time and never got any problem with Opera and FF. 
I'm always using it with local file I never use CND.
I think you should try to use all libraries locally instead using CDN this may fix the problems.

**Comment by kressly** [comment]
@lenamtl
Locally it works in Every browser (Firefox, Ie, Opera,  Safari and Chrome)
 BUT on Opera on android Phone, 
```
<i class="fa fa-envelope"></i>
```
shows nothing. So unless the visitor  click on the empty space, he may never know that there is a link there that leads to the messages.
I have tested it on two different android.

**Comment by lenamtl** [comment]
check this:
https://github.com/FortAwesome/Font-Awesome/issues/791
http://stackoverflow.com/questions/15028602/font-awesome-icons-are-not-working-in-some-browsers/15256010#15256010

**Comment by almasaeed2010** [comment]
@kressly please let us know which Android version you are using?
It's not easy to get everything working on every single browser. AdminLTE provides support to the latest versions of all major browsers except in the case of IE which it supports IE9 up to 11. 

I hope the problem doesn't persist with the latest version of Opera Mobile on Android. I tested it on iOS and it worked fine. In case it persists, please let us know! 

Thanks to all for the contributions and compliments.

**Comment by lenamtl** [comment]
Ok I have found out that opera mini browser doesn't support web font.
https://github.com/FortAwesome/Font-Awesome/issues/4565

**Comment by kressly** [comment]
Oh ok. I guess the default browser of that android 4.0.4 ans Opera both do not support it. 
Now also, there is a white skin that is in the package and i like it a lot but @almasaeed2010 did not include it in the demo among the skin. It should be . It is very nice.

**Comment by lenamtl** [comment]
ok good to know I didn't see it, I prefer light and clean skin

**Comment by almasaeed2010** [comment]
Hello

@kressly which skin is that? I don't recall any skins that are not in the demo.

There are 2 extra skins coming soon that won't use the dark sidebar. Also, since skins are not light in size, they will be removed from the main CSS and compiled to have their own css files that should be included separately depending on the skin you prefer. This way there isn't going to be a down side in having so many skins.

**Comment by kressly** [comment]
Ok i checked again and actually it was a  BUG in the Black skin that generated in a WHITE skin and i wish we can include a white skin like that.
When you download http://almsaeedstudio.com/themes/AdminLTE/AdminLTE2-dist.zip and click on Black skin, you get an awesome White skin
![1](https://cloud.githubusercontent.com/assets/925132/6050928/8b653504-acbb-11e4-9ee3-6cabfb0d452b.png)
I don't know what caused that bug but i like that bug. If we can have a white skin like that , really i think it is good.

**Comment by kressly** [comment]
Another Bug is that the left menu bar does not disappear on link click. For example, i tried to read the upgrade documentation on my phone and after clicking on the link, i had to click again on the menu bar for it to collapse so that i may read. A normal user may never do that. He may be waiting and not even know that the page has moved to the anchor. So they should be a way of making the nav bar disapear on click and we will be left with the choice of making it disapear or not

**Comment by yajra** [comment]
[BUG] on printing. Left menu width is still occupying the print layout although it's not visible. Version 1 only displays the content area as printable. Thanks!

**Comment by jems1234** [comment]
hey.. Can you make  AdminLTE IE6+ compatible I can pay 50$ for this changes... Please reply.

**Comment by almasaeed2010** [comment]
Solved

**Comment by kressly** [comment]
@almasaeed2010 
Please what have been solved ?

**Comment by lenamtl** [comment]
We should check to see is all scrollbar issues are solved.

**Comment by almasaeed2010** [comment]
I closed this by mistake last night! I thought it was a separate issue to make AdminLTE IE6+ compatible which I have addressed with @jems1234 on facebook. To summarize, I don't think it's possible. 

@kressly  to see major changes, you can view the changelog in the master repo. To view every change you can go through the commits.

@lenamtl we already have too many css files so creating an extra one for a few lines of code doesn't seem appealing. However, we can separate the printing rules in the less files (create print.less). This way we can allow developers to modify the grunt.js and create the css files the want. I checked the scrollbar issues on Windows IE9, which was causing the most trouble, and it didn't show. Nonetheless, I'd appreciate it if you checked in your devices too. 

The benefit of having alpha and beta versions is to get as much tested as possible before publishing for production environments.

**Comment by thomsol** [comment]
Great Template!

I'm anxiously awaiting the release, but...   If I select Fixed Layout, the left sidebar menu is no longer scrollable (it used to scroll separately).  Thus, if expanded, there are some menu items which you cannot get to.

**Comment by almasaeed2010** [comment]
This issue is getting very long and annoying to scroll through. I would greatly to appreciate it if you guys opened new separate issues for any problem. I'd like to make it easier to follow. Addressing each issue separately seems like the best option. This issue should be closed, don't you think?

**Comment by almasaeed2010** [comment]
@thomsol Thanks! To address your issue, in AdminLTE 2.0 you must load the slimScroll plugin separately. Please make sure that you have done that. If the issue persists, please open a new issue for it. I appreciate your contribution.

**Comment by fredrkr** [comment]
Looks really good!

I would, however, suggest moving away from Morris.js for charts and over to another relatively light-weight chart-library, like Chart.js (http://www.chartjs.org) for instance. The author of Morris.js is no longer maintaining the library. Now, granted, it still works, but wouldn't it be safer to go for a library for which bugs are getting fixed at least?

Thanks soooo much for this awesome theme!

**Comment by kressly** [comment]
@fredrkr i checked chartjs.org and it is awesome. However like @almasaeed2010 said, please open a new issue named AdminLte 2 Chart so that we can participate from there. Can we close this issue please ?

**Comment by almasaeed2010** [comment]
@fredrkr That's a great suggestion! This is exactly what we did for [Dashboard v2](http://almsaeedstudio.com/themes/AdminLTE/index2.html). We loved ChartJS once we saw it.

@kressly you are right. It would be preferable if new topics are opened as separate issues.

**Comment by lukacavic** [comment]
Hello, amazing template, using it everywhere :)

on tip or sugestion....I saw in other temoplates use of right sidebar!

On click it opens the right sidebar!

![untitled](https://cloud.githubusercontent.com/assets/4538071/6329785/bccfc7f0-bb73-11e4-8007-4590ea3e718f.png)
